frame/contracts/fixtures/sr25519_verify.wat

```diff
@@ -0,0 +1,55 @@
+;; This contract:
+;; 1) Reads signature, message and public key from the input
+;; 2) Calls and return the result of sr25519_verify
+
+(module
+    ;; import the host functions from the seal0 module
+	(import "seal0" "seal_sr25519_verify" (func $seal_sr25519_verify (param i32 i32 i32 i32) (result i32)))
+	(import "seal0" "seal_input" (func $seal_input (param i32 i32)))
+	(import "seal0" "seal_return" (func $seal_return (param i32 i32 i32)))
+
+	;; give the program 1 page of memory
+	(import "env" "memory" (memory 1 1))
+
+	;; [4, 8) length of signature + message + public key - 64 + 12 + 32 = 108 bytes
+	;; write the length of the input (108 bytes) at offset 4
+	(data (i32.const 4) "\6c")
```

Comment:
`0` is a valid address in wasm. Maybe I am missing something but I can't see a reason to start at `4` rather than zero.